server/portal/settings/settings.py

```diff
@@ -557,7 +557,7 @@ def gettext(s): return s  # noqa:E731
     'tag_enabled': True,
 }
 
-AGAVE_DEFAULT_TRASH_NAME = '.Trash'
+TAPIS_DEFAULT_TRASH_NAME = '.Trash'
```

Comment:
It just occurred to me that this `TAPIS_DEFAULT_TRASH_NAME ` setting is hardcoded here, and the setting in `settings_custom` is unused. Could you refactor this bit to be
```suggestion
TAPIS_DEFAULT_TRASH_NAME = getattr(settings_custom, '_TAPIS_DEFAULT_TRASH_NAME', '.Trash')
```